settings.gradle

```diff
@@ -0,0 +1,16 @@
+pluginManagement {
```

Comment:
needs license, you can see examples here: https://github.com/android/compose-samples/find/main